maintainers/maintainer-list.nix

```diff
@@ -15197,6 +15197,12 @@
     githubId = 30825096;
     name = "Ning Zhang";
   };
+  o0th = {
+    email = "o0th@pm.me";
+    name = "Sabato Luca Guadagno";
+    github = "o0th";
+    githubId = 22490354;
+  };
```

Comment:
I forgot to mention that this needs to be a separate commit: `maintainers: add <handle>`